src/node_report.cc

```diff
@@ -300,7 +302,8 @@ static void PrintVersionInformation(JSONWriter* writer) {
   }
 
   PrintCpuInfo(writer);
-  PrintNetworkInterfaceInfo(writer);
+  if (!disable_network)
```

Comment:
We should add a test explicitly for this change

Reply:
Inside of `test/cctest/test_report.cc`?